src/components/pages/History/PublishedList.tsx

```diff
@@ -4,33 +4,60 @@ import { useOcean } from '@oceanprotocol/react'
 import React, { ReactElement, useEffect, useState } from 'react'
 import Loader from '../../atoms/Loader'
 import AssetList from '../../organisms/AssetList'
+import axios from 'axios'
+import { queryMetadata } from '../../../utils/aquarius'
 
 export default function PublishedList(): ReactElement {
-  const { ocean, status, accountId } = useOcean()
+  const { accountId } = useOcean()
   const [queryResult, setQueryResult] = useState<QueryResult>()
   const [isLoading, setIsLoading] = useState(false)
+  const { config } = useOcean()
+  const source = axios.CancelToken.source()
+  const [page, setPage] = useState<number>(1)
 
   useEffect(() => {
     async function getPublished() {
-      if (!accountId || !ocean) return
-
+      if (!accountId) return
+      const queryPublishedAssets = {
+        page: page,
+        offset: 9,
```

Comment:
good thinking migrating `ocean.assets.ownerAssets` to Aquarius query. If we can't easily deal with the layout collapse on page change, I would then limit the displayed items to `6` here so only 2 rows on desktop are shown at once. Just slightly improves the layout collapse

Reply:
I found a solution for that, basically I don't show the loading icon between page changes so It doesn't collapse. Only show the loading on page mount. Not sure if trading the loading icon for that is a good idea, what do you think?

Reply:
ah, sorry, should have checked out preview, this is great as is!